cerebrium/fine-tuning/introduction.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Introduction"
+description: "Quickly and conveniently fine-tune your LLMs on with just one line of code"
+---
+
+<Note>Cerebrium's fine-tuning functionality is in public beta and so we are adding more functionality each week! Currently we only support the 
+training of CasualLM models. If you have a urgent requirement, we can help you just reach out to [support](mailto\:support@cerebrium.ai)</Note>
+
+
+The fine-tuning functionality on Cerebrium allows you to quickly and conveniently fine-tune your LLMs on Cerebrium with just one line of code. Cerebrium leverages
+the latest techniques such as PEFT and LoRA in order to train models in order to do so in the shortest amount of time (and therefore cost) while still achieving the
+same performance. 
+
+Currently, our fine-tuning capabilities are limited to any causal language models from the HuggingFace transformers library. Some of these models include:
```

Comment:
Minor correction: Any causal language model that supports 8bit quantisation/lora